pkgs/tools/inputmethods/ibus-engines/ibus-mozc/default.nix

```
@@ -21,32 +22,22 @@ in clangStdenv.mkDerivation rec {
     maintainers  = with maintainers; [ gebner ericsagnes ];
   };
 
-  nativeBuildInputs = [ which ninja python gyp pkgconfig qt5.wrapQtAppsHook ];
+  nativeBuildInputs = [ which ninja python python37Packages.six gyp pkgconfig qt5.wrapQtAppsHook ];
```

```suggestion
  nativeBuildInputs = [ which ninja python3Packages.python python3Packages.six gyp pkgconfig qt5.wrapQtAppsHook ];
```
Otherwise the versions of the python interpreter and the six module could be different.